### ISSUE TITLE
Feature Request

### ISSUE THREAD

**Issue by f3l1p1nh07** [PocketMobsters/MobBotTemp#114]
1 - Add option to not filter considering the CP only IV and / or level. example: if set CP: 0 (disregarding the CP) only consider the IV / LEVEL.

2 - Add option of operation to analyze whether the Pokemon has the listed moves and minimum requirements example:

   "Snorlax": {
       "KeepMinCp": 6000,
       "KeepMinLvl" 23
       "UseKeepMinLvl": true,
       "KeepMinIvPercentage": 90.0,
       "KeepMinDuplicatePokemon": 3,
       "Moves": [
         [
           "ZenHeadbuttFast"
           "Bodyslam"
         ]
       ]
       "DeprecatedMoves": []
       "KeepMinOperator": "or"
       "MovesOperator": "and"
       "MovesKeepOperator": "and" * NEW FUNCTION *
     }

put a MovesKeepOperator filter = "and" or "or"

if set to "and" the pokemon should contain the listed moves, and cp / iv / level listed otherwise will be transferred, if set to "or" when he has any of the requirements it does not transfer.

**Comment by f3l1p1nh07** [comment]
@tangdami  right!, So how do you bot ignore the CP and consider only IV and Level?

**Comment by tangdami** [comment]
At least there is no suck an option for MobBot :(
Other bots have it, I think you were quoting Necro? I also set it as minimum CP 100, Lvl 25 and IV 95% (operator is "and"). But my second account was banned, I think Necro lacks settings of randomization and humanized patching were the reason. So I switched back to MobBot ;)

**Comment by ridalarry** [comment]
I would love to see this added
